### DISCUSSION TITLE
The new azure developer cli template (todo-csharp-sql-swa-func), doesn't seem to rebuild when making changes to tsx files. Is my assumption correct that the typescript files are supposed to compile upon saving and then propagate to the localhost server and then refresh in the browser?

### DISCUSSION THREAD

**Discussion by jalmarez** [Azure/azure-dev#2492] [Q&A]
The new azure developer cli template (todo-csharp-sql-swa-func), doesn't seem to rebuild when making changes to tsx files. Is my assumption correct that the typescript files are supposed to compile upon saving and then propagate to the localhost server and then refresh in the browser?

**Comment by savannahostrowski** [comment]
Hey @jalmarez - are you running one of the tasks from the launch.json in VS Code (e.g. "Start Web") or `npm run start`? That should work as you describe. You'd just need to refresh the page to see changes.

**Comment by jalmarez** [comment]
I am using the Start Web task but making any changes to a tsx page, layout
or component does not reflect in the browser unless I stop the process and
restart the Start Web task. I’m using the Dev Container approach on windows
through docker.
